### PR TITLE
Duplicate core's approach of wrapping script embeds in iframe

### PR DESCRIPTION
**Pull request by danielbachhuber**
See #28

### CODE REVIEW

**Comment by danielbachhuber** [comment]
Boomski:

image

File: css/shortcode-ui-editor-styles.css.map [hunk 1]
```diff
@@ -3,5 +3,5 @@
   "file": "",
   "sources": ["sass/shortcode-ui-editor-styles.scss"],
   "names": [],
-  "mappings": "AAAA;EACC,YAAY"
+  "mappings": "AAAA;EACC,YAAY;EAEb,YAAY,0BAA0B;IAGnC,SAAS"
```
Comment (davisshaver) [inline css/shortcode-ui-editor-styles.css.map:6]: What's this?

**Comment by davisshaver** [comment]
@danielbachhuber, this looks good to me. Only concern I have is replicating core functionality. Could you explain why we're doing this?

**Comment by danielbachhuber** [comment]
> Only concern I have is replicating core functionality. Could you explain why we're doing this?

To clarify, here's core's existing functionality: https://core.trac.wordpress.org/browser/tags/4.0/src/wp-includes/js/mce-view.js#L126

However, the `setIframes` function is more closely coupled to the rest of the model, including fetching the preview and rendering. Our model for fetching the preview and rendering is separate, which means we'd have to refactor some to make it work.

The API is marked as experimental though, so I don't want to create a dependency that's going to break. In short, let's internalize for now so we can better ensure stability. It can be refactored later if it makes it into core.

File: css/shortcode-ui-editor-styles.css.map (hunk 1 above)
Comment (danielbachhuber) [inline css/shortcode-ui-editor-styles.css.map:6]: Junk stuff

**Pull request merged**